master/custom/factories.py

```diff
@@ -369,3 +369,17 @@ class Windows64ICCReleaseBuild(Windows64ReleaseBuild):
     buildersuffix = ".nondebug"
     buildFlags = Windows64ReleaseBuild.buildFlags + windows_icc_build_flags
     factory_tags = ["win64", "icc", "nondebug"]
+
+class WindowsArm32Build(WindowsBuild):
+    buildFlags = ["-p", "ARM", "--no-tkinter"]
+    # test_multiprocessing_spawn doesn't complete over simple ssh connection
+    # skip test_multiprocessing_spawn for now
+    testFlags = ["-arm32", "-j2", "-x", "test_multiprocessing_spawn", "-x", "test_winconsoleio", "-x", "test_distutils"]
+    cleanFlags = ["-p", "ARM", "--no-tkinter"]
+    factory_tags = ["arm32"]
```

Comment:
We need some manner of Windows tag here too.  Maybe `win-arm32` instead of `arm32`, or a separate `windows` tag?  Same below.